docs/src/components/componentList/drawerBodyDocsComponent/DrawerBodyDocsComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Once you the base branch to resolve merge conflicts, this component file won't be needed and you can directly use DrawerBodyMarkdown in the navigation.

- [ ] Use DrawerBodyMarkdown directly in navigation

Reply:
Not sure what you mean. You have updated the react-dev-docs? The merge conflict is here...

![image](https://user-images.githubusercontent.com/37150565/181745166-627997ef-04a7-4ce3-9461-acfd26720777.png)